ics/v9/views.py

```
@@ -955,3 +963,54 @@ def get(self, request):
 
     return Response(objects)
 
+
+###################################
+# RECIPES RELATED VIEWS #
+###################################
+
+class RecipeList(generics.ListCreateAPIView):
+  queryset = Recipe.objects.all()
+  serializer_class = RecipeSerializer
+  filter_fields = ('product_type', 'process_type', 'id')
+
+  def get_queryset(self):
+    team = self.request.query_params.get('team', None)
+    if team is not None:
```

In general, I think we should return an error if `team is None` in order to prevent showing users data from a different team.